css/map.css

```diff
@@ -613,21 +617,21 @@ noscript p
 #locateButton
 {
 	position: absolute;
-	left: 310px;
-	top: 70px;
-	width: 26px;
-	height: 26px;
+	left: 311px;
+	top: 82px;
+	width: 30px;
+	height: 30px;
 	background-color: white;
 	background-image: url('../img/locate.png');
 	background-repeat: no-repeat;
 	background-position: 50% 50%;
 	background-size: 23px 23px;
-	-moz-border-radius: 4px;
-	-webkit-border-radius: 4px;
-	-o-border-radius: 4px;
+	-moz-border-radius: 2px;
+	-webkit-border-radius: 2px;
+	-o-border-radius: 2px;
 	border-radius: 4px;
-	border: 1px solid #aaa;
-	box-shadow: 0 1px 7px rgba(0,0,0,0.4);
+	border: 2px solid rgba(0,0,0,0.2);
+	background-clip: padding-box;
```

Comment:
The changes are related because the upgrade of the Leaflet library causes the button to look slightly different. This style for the location button is copying the Leaflet button style, so it should be adapted to look the same way.

(current buttons)
![image](https://user-images.githubusercontent.com/1073881/210649705-5844c6ed-fe68-41a5-8986-f1bacbd5f9af.png)

(buttons in the new leaflet without changes to the location button)

![image](https://user-images.githubusercontent.com/1073881/210650659-115cbb69-ba5d-4ee9-962c-09de5eb7f738.png)


(buttons in the new leaflet with changes to the location button)
![image](https://user-images.githubusercontent.com/1073881/210649964-0f3887d0-b5ee-46af-94ff-f7b423316ef9.png)


It would be better to use a native Leaflet control for the location button, instead of manually positioning the button on top of the map, but I didn't want to touch too many different things in one PR.

Reply:
Then it is fine.

Reply:
Alternatively, instead of imitating the Leaflet styles, we could _use_ the already loaded styles. See #885 for this.